### PR TITLE
Apply TorchFix TOR203 fixes

### PR DESCRIPTION
**Pull request by kit1980**
Codemodded via `torchfix . --select=TOR203 --fix`.
This is a step to unblock https://github.com/pytorch/pytorch/pull/141076

### CODE REVIEW

**Review by malfet — APPROVED** [review]

File: functorch/examples/dp_cifar10/cifar10_transforms.py [hunk 1]
```diff
@@ -12,8 +12,7 @@
 from datetime import datetime, timedelta
 
 import numpy as np
-import torchvision.transforms as transforms
```
[the remark this answers is not in the document]
Comment (kit1980) [reply functorch/examples/dp_cifar10/cifar10_transforms.py:15]: I talked to people some time ago, and `import torchvision.transforms as transforms` (same for `models`) seems to be an artifact everybody copied from a very old TorchVision tutorial.

**Pull request closed without merging**